dotnet/samples/Concepts/ChatCompletion/OpenAI_ChatCompletionStreaming.cs

```diff
@@ -32,24 +41,98 @@ public Task AzureOpenAIChatStreamSampleAsync()
         return this.StartStreamingChatAsync(chatCompletionService);
     }
 
+    /// <summary>
+    /// This example demonstrates how the chat completion service streams text content.
+    /// It shows how to access the response update via StreamingChatMessageContent.Content property
+    /// and alternatively via the StreamingChatMessageContent.Items property.
+    /// </summary>
+    [Fact]
+    public async Task StreamTextContentAsync()
```

Comment:
Why is this sample structured differently from the previous two?

Reply:
The previous two examples show the streaming concept for both OpenAI and AzureOpenAI connectors, even though the streaming code is 99% the same. This sample and the next one demonstrate streaming text and function content without repeating it twice—once per connector. I'm happy to add four more methods: two public ones as tests and two private ones shared by the public methods. Alternatively, we can restructure the samples above.

Reply:
Decided to resolve it later.